molecule/in/tests/test_ingest.py

```diff
@@ -3,7 +3,24 @@
 """Testing module for ingest to verify working."""
 
 
-def test_ingest(host):
-    """Test the ingest default port."""
+def test_ingest_socket(host):
+    """Test the ingest default socket."""
     sock = host.socket('tcp://0.0.0.0:8066')
     assert sock
+
+
+def check_ingest_backend_service(host):
+    """Check that the ingest backend service is running on the host."""
+    assert host.service('ingest_backend').is_running
+
+
+def check_ingest_frontend_service(host):
+    """Check that the ingest frontend service is running on the host."""
+    assert host.service('ingest_frontend').is_running
+
+
+# def test_ingest_return(host):
```

Comment:
Commenting out tests is kinda lame... The testing is run by PyTest and we should use the skipping methods they provide. http://doc.pytest.org/en/latest/skipping.html

Reply:
So example code...

```
from pytest.mark import skip

@skip(reason="no way of currently testing this")
```

just before this line.

Reply:
I thought we were using `testinfra`? Besides that, maybe you could shed light on why my ability to curl root of each service for ingest, cartd, and notifications might have been returning 500? The service(s) seem to be running and the system(s) are listening on the appropriate ports.. Mind taking a look? It thought the versions being deployed would have that api test response available but even in my production system it doesn't seem to be working as described in the cartd, ingest, and notifications repo testing.. If this can be solved we can reenable these tests, I just dint want to lose the goal to enable them in some way. Combining pytest with testinfra should work I was just thinking it may be better to figure out why it is breaking just in these cases and not say core or archiveinterface scenarios.

Reply:
Just running my local tests then I should have the pytest bit pushed out.